crates/bevy_render/src/batching/gpu_preprocessing.rs

```diff
@@ -935,16 +935,21 @@ impl FromWorld for GpuPreprocessingSupport {
             crate::get_adreno_model(adapter).is_some_and(|model| model != 720 && model <= 730)
         }
 
-        let max_supported_mode = if device.limits().max_compute_workgroup_size_x == 0 ||
-            is_non_supported_android_device(adapter)
+        let feature_support = device.features().contains(
+            Features::INDIRECT_FIRST_INSTANCE
+                | Features::MULTI_DRAW_INDIRECT
+                | Features::PUSH_CONSTANTS,
+        );
+        let limit_support = device.limits().max_storage_textures_per_shader_stage >= 12;
```

Comment:
Maybe factor the 12 out into a const with a comment?

Reply:
Added a comment